docs/source/en/quicktour.mdx

```diff
@@ -14,83 +14,57 @@ specific language governing permissions and limitations under the License.
 
-## Pipeline
+Before you begin, make sure you have 🤗 Transformers installed:
 
-[`pipeline`] is the easiest way to use a pretrained model for a given task.
+```bash
+pip install transformers
+```
 
-<Youtube id="tiZFewofSLM"/>
+You'll also want to install 🤗 Datasets to load a dataset to train on quickly:
 
-The [`pipeline`] supports many common tasks out-of-the-box:
+```bash
+pip install datasets
+```
 
-**Text**:
-* Sentiment analysis: classify the polarity of a given text.
-* Text generation (in English): generate text from a given input.
-* Name entity recognition (NER): label each word with the entity it represents (person, date, location, etc.).
-* Question answering: extract the answer from the context, given some context and a question.
-* Fill-mask: fill in the blank given a text with masked words.
-* Summarization: generate a summary of a long sequence of text or document.
-* Translation: translate text into another language.
-* Feature extraction: create a tensor representation of the text.
+## Pipeline
 
-**Image**:
-* Image classification: classify an image.
-* Image segmentation: classify every pixel in an image.
-* Object detection: detect objects within an image.
+The [`pipeline`] is the easiest way to use a pretrained model for inference. You can use the [`pipeline`] out-of-the-box for many tasks such as text generation, image classification, automatic speech recognition, and many more. 
 
-**Audio**:
-* Audio classification: assign a label to a given segment of audio.
-* Automatic speech recognition (ASR): transcribe audio data into text.
+<Youtube id="tiZFewofSLM"/>
 
 <Tip>
 
-For more details about the [`pipeline`] and associated tasks, refer to the documentation [here](./main_classes/pipelines).
+For more details about tasks supported by [`pipeline`], take a look at the [pipelines API reference](./main_classes/pipelines).
 
 </Tip>
 
-### Pipeline usage
-
-In the following example, you will use the [`pipeline`] for sentiment analysis.
-
-Install the following dependencies if you haven't already:
-
-<frameworkcontent>
-<pt>
-```bash
-pip install torch
-```
-</pt>
-<tf>
-```bash
-pip install tensorflow
-```
-</tf>
-</frameworkcontent>
-
-Import [`pipeline`] and specify the task you want to complete:
+To use a [`pipeline`], create an instance of it and specify a task you want to complete, like sentiment analysis, for example:
 
 ```py
 >>> from transformers import pipeline
 
 >>> classifier = pipeline("sentiment-analysis")
 ```
 
-The pipeline downloads and caches a default [pretrained model](https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english) and tokenizer for sentiment analysis. Now you can use the `classifier` on your target text:
+For sentiment analysis, the [`pipeline`] downloads and caches a default [pretrained model](https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english) and tokenizer. Now you can use the `classifier` on your target text:
```

Comment:
This gives the impression that this behavior is specific to the sentiment analysis task, when it's general. So the previous phrasing was better in that way.